core/base/src/TUUID.cxx

```diff
@@ -21,6 +21,14 @@ originally used in the Network Computing System (NCS) and
 later in the Open Software Foundation's (OSF) Distributed Computing
 Environment (DCE).
 
+Note: In the way this UUID is constructed, when used outside of
```

Comment:
```suggestion
\note In the way this UUID is constructed, when used outside of
```
This would make it look like the yellow `Note` section [here](https://root.cern.ch/doc/master/classRooAbsPdf.html)